analysis/src/CompletionFrontEnd.ml

```diff
@@ -1324,10 +1322,29 @@ let completionWithParser1 ~currentFile ~debug ~offset ~path ~posCursor
                         inJsx = !inJsxContext;
                       }))
             | None -> ())
-        | Pexp_apply {funct = {pexp_desc = Pexp_ident compName}; args}
-          when Res_parsetree_viewer.is_jsx_expression expr ->
+        | Pexp_jsx_element
+            ( Jsx_unary_element
```

Comment:
thoughts about pros/cons of unifying the jsx ast nodes into one?
In this case, perhaps one could streamline the code a bit more, but don't know about the rest

Reply:
I believe combining these is beneficial, although it may not be a game changer. If you don't need to handle JSX, you can avoid a single branch in your match. However, if you do need to handle them, you'll need to address all of them.